back/spec/acceptance/initiatives_spec.rb

```diff
@@ -672,6 +673,20 @@
           expect(json_response_body.dig(:errors, :base, 0, :error)).to eq 'Unauthorized!'
         end
       end
+
+      describe 'cosponsor_ids' do
+        let(:id) { @initiative.id }
+        let(:cosponsor) { create(:user) }
+        let(:cosponsor_ids) { [cosponsor.id] }
+
+        example 'Update the cosponsors of an initiative', document: false do
```

Comment:
```suggestion
        example 'Update the cosponsors of an initiative' do
```
Can it be useful for the docs?

Reply:
Ah yes, clearly. Thanks!